tests/dataset_test.py

```diff
@@ -0,0 +1,141 @@
+from qri import dataset
+from qri import loader
+import mock_loader
+import unittest
+
+# TODO - move these to a better place?
+LIST_OBJ = {
+    'username': 'me',
```

Comment:
The username "me" is actually disallowed, it's only a UX convenience, but shouldn't be returned by a list operation. Usually for tests we use "peer", could you make that change?